astroquery/vizier/tests/test_vizier_remote.py

```diff
@@ -9,6 +9,8 @@
 from astroquery.utils import commons
 from .conftest import scalar_skycoord, vector_skycoord
 
+vizier.Vizier.clear_cache()
```

Comment:
I'm not sure how likely it is to affect users, I would think whoever wants to run remote tests should be OK with clearing their caches...

Reply:
Just me being a bit lazy while running the tests. Removing it :wink: